aviary/subsystems/propulsion/turboprop_model.py

```diff
@@ -79,6 +107,16 @@ def build_pre_mission(self, aviary_inputs, **kwargs) -> om.Group:
                     promotes=['*']
                 )
 
+        gearbox_model_pre_mission = gearbox_model.build_pre_mission(
+            aviary_inputs, **kwargs
+        )
+        if gearbox_model_pre_mission is not None:
```

Comment:
Will this leave mass.gearbox at zero if pre-mission isn't initialized?

Reply:
That's my understanding. If a custom gearbox model has no pre-mission and the user doesn't override a mass value in the input file, then mass _should_ stay at the default of zero, which is hopefully expected/intuitive behavior. I don't have a test of that though